backend/src/main/java/gov/cdc/usds/simplereport/config/GraphQlConfig.java

```diff
@@ -61,9 +71,14 @@ public DataFetcherExceptionResolver dataFetcherExceptionResolver() {
       }
 
       if (exception instanceof IllegalGraphqlArgumentException) {
-        String errorMessage =
-            String.format("header: %s; %s", exception.getMessage(), defaultErrorBody);
-        return Mono.just(singletonList(new GenericGraphqlException(errorMessage, errorPath)));
+        if ("addUserToCurrentOrg".equals(errorPath)
```

Comment:
Rather than catching and then reformatting this exception, could you instead go to the source of the error and throw a different type of exception there? Potentially throwing a `ConflictingUserException` instead of an `IllegalGraphQlArgumentException` [here](https://github.com/CDCgov/prime-simplereport/blob/89a5a7bbe4a5c6442fe7cb32e0bce97b61e0f8be/backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java#L277-L277)?

If possible, it's best practice to keep exceptions (and their error messages) as close to the source of the problem as possible in code - otherwise, we risk someone changing the original thrown exception in `LiveOktaRepository` but not realizing they need to change this `if` statement in order to have the correct error message shown to the user.